catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models/redeliveryPolicy.json

```diff
@@ -17,12 +17,12 @@
-    "useExponentialBackOff": { "index": 5, "kind": "attribute", "displayName": "Use Exponential Back Off", "label": "advanced", "required": false, "type": "boolean", "javaType": "java.lang.Boolean", "deprecated": false, "autowired": false, "secret": false, "defaultValue": false, "description": "Turn on exponential backk off" },
+    "useExponentialBackOff": { "index": 5, "kind": "attribute", "displayName": "Use Exponential Back Off", "label": "advanced", "required": false, "type": "boolean", "javaType": "java.lang.Boolean", "deprecated": false, "autowired": false, "secret": false, "defaultValue": false, "description": "Turn on exponential back off" },
     "collisionAvoidanceFactor": { "index": 6, "kind": "attribute", "displayName": "Collision Avoidance Factor", "label": "advanced", "required": false, "type": "number", "javaType": "java.lang.Double", "deprecated": false, "autowired": false, "secret": false, "defaultValue": "0.15", "description": "Sets the collision avoidance factor" },
     "useCollisionAvoidance": { "index": 7, "kind": "attribute", "displayName": "Use Collision Avoidance", "label": "advanced", "required": false, "type": "boolean", "javaType": "java.lang.Boolean", "deprecated": false, "autowired": false, "secret": false, "defaultValue": false, "description": "Turn on collision avoidance." },
     "maximumRedeliveryDelay": { "index": 8, "kind": "attribute", "displayName": "Maximum Redelivery Delay", "required": false, "type": "duration", "javaType": "java.lang.String", "deprecated": false, "autowired": false, "secret": false, "defaultValue": "60000", "description": "Sets the maximum delay between redelivery" },
-    "retriesExhaustedLogLevel": { "index": 9, "kind": "attribute", "displayName": "Retries Exhausted Log Level", "label": "advanced", "required": false, "type": "object", "javaType": "org.apache.camel.LoggingLevel", "deprecated": false, "autowired": false, "secret": false, "defaultValue": "ERROR", "description": "Sets the logging level to use when retries have been exhausted" },
-    "retryAttemptedLogLevel": { "index": 10, "kind": "attribute", "displayName": "Retry Attempted Log Level", "required": false, "type": "object", "javaType": "org.apache.camel.LoggingLevel", "deprecated": false, "autowired": false, "secret": false, "defaultValue": "DEBUG", "description": "Sets the logging level to use for logging retry attempts" },
+    "retriesExhaustedLogLevel": { "index": 9, "kind": "attribute", "displayName": "Retries Exhausted Log Level", "label": "advanced", "required": false, "type": "enum", "javaType": "org.apache.camel.LoggingLevel", "enum": [ "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "OFF" ], "deprecated": false, "autowired": false, "secret": false, "defaultValue": "ERROR", "description": "Sets the logging level to use when retries have been exhausted" },
```

Comment:
You changed the default for `retriesExhaustedLogLevel` from `DEBUG` to `ERROR`. Is that intended?

Reply:
ah I see the difference to https://github.com/apache/camel/pull/13312/files#diff-df860c9ccdcd78a71b9e096d18c329d0559d700d40c4cda5be053025444df3cf so this seems fine then.